moped-editor/src/components/GridTable/GridTableFiltersCommonOperators.js

```diff
@@ -29,24 +29,40 @@ export const GridTableFiltersCommonOperators = {
   string_is_not_null: {
     operator: "_is_null",
     label: "is not blank",
-    description: "String field is not null",
+    description: "Selected field is not null",
     envelope: "false",
     type: "string",
   },
+  string_is_blank: {
+    operator: "_is_null",
+    label: "is blank",
+    description: "Selected field is blank",
+    envelope: "true",
+    specialNullValue: '""',
+    type: "string",
+  },
+  string_is_not_blank: {
+    operator: "_is_null",
+    label: "is not blank",
+    description: "Selected field is not blank",
+    envelope: "false",
+    specialNullValue: '""',
+    type: "string",
+  },
   string_is_null_special_case: {
     operator: "_is_null",
     label: "is blank",
     description: "Selected field does not have meaningful content",
     envelope: "true",
-    specialNullValue: '" :"',
+    specialNullValue: '"None"',
     type: "string",
   },
   string_is_not_null_special_case: {
     operator: "_is_null",
     label: "is not blank",
     description: "String field does not match special null value",
     envelope: "false",
-    specialNullValue: '" :"',
+    specialNullValue: '"None"',
```

Comment:
Given that there is a way to handle a `specialNullValue`, this isn't a blocker by any means, but we may want to consider unifying all the columns' behavior. I think when there is a representation of the null set of something, we have the DB return`null`. If we want to show "None" in the UI, no problem, but the actual data we get back, we may want to conform to a agreed upon way we do things across the board.